From d9f030a32a1a445f404d616ce3b5df96fb5be1a6 Mon Sep 17 00:00:00 2001
From: Ken Chalmers <ken.chalmers@amd.com>
Date: Thu, 19 May 2016 11:08:12 -0400
Subject: [PATCH 0497/1722] drm/amd/dal: remove const from enum return type

"const enum" as a return type results in "warning: type qualifiers
ignored on function return type".

Signed-off-by: Ken Chalmers <ken.chalmers@amd.com>
Acked-by: Jordan Lazare <Jordan.Lazare@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc.c | 2 +-
 drivers/gpu/drm/amd/dal/dc/dc.h      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc.c b/drivers/gpu/drm/amd/dal/dc/core/dc.c
index ed990ac..daaa705 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc.c
@@ -1148,7 +1148,7 @@ const struct ddc_service *dc_get_ddc_at_index(
 	return core_dc->links[link_index]->ddc;
 }
 
-const enum dc_irq_source dc_get_hpd_irq_source_at_index(
+enum dc_irq_source dc_get_hpd_irq_source_at_index(
 	struct dc *dc, uint32_t link_index)
 {
 	struct core_dc *core_dc = DC_TO_CORE(dc);
diff --git a/drivers/gpu/drm/amd/dal/dc/dc.h b/drivers/gpu/drm/amd/dal/dc/dc.h
index 7ec68a6..09725fa 100644
--- a/drivers/gpu/drm/amd/dal/dc/dc.h
+++ b/drivers/gpu/drm/amd/dal/dc/dc.h
@@ -516,7 +516,7 @@ enum dc_irq_source dc_interrupt_to_irq_source(
 		uint32_t ext_id);
 void dc_interrupt_set(const struct dc *dc, enum dc_irq_source src, bool enable);
 void dc_interrupt_ack(struct dc *dc, enum dc_irq_source src);
-const enum dc_irq_source dc_get_hpd_irq_source_at_index(
+enum dc_irq_source dc_get_hpd_irq_source_at_index(
 		struct dc *dc, uint32_t link_index);
 const struct dc_target *dc_get_target_on_irq_source(
 		const struct dc *dc,
-- 
2.7.4

